### PR TITLE
S2U-21 SEB: Improve settings info, fix nullpointer and align buttons

### PR DESCRIPTION
**Pull request by stetsche**
As @mpellicer pointed out on https://sakaiproject.atlassian.net/browse/S2U-21 after additional following issues were found:
1. There was a null pointer when publishing the assessment from the assessment list -> Actions -> Edit -> Publish
2. More info in the settings about the SEB config mode would be helpful
3. Buttons in student assessment view are not aligned 

Those issues are addressed by this PR.

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties [hunk 1]
```diff
@@ -379,6 +379,13 @@ seb_config_mode=Com vols configurar Safe Exam Browser?
 seb_config_mode_client=Usar la configuraci\u00f3 del client
 seb_config_mode_manual=Configurar manualment
 seb_config_mode_upload=C\u00e0rrega de configuraci\u00f3 pr\u00f3pia
+seb_config_mode_client_help=Using client configuration will allow the use of encrypted or no SEB configuration. In this case no \
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties:382]: @mpellicer Could you add _ca translation please?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties [hunk 2]
```diff
@@ -420,6 +420,13 @@ seb_config_mode=\u00bfC\u00f3mo quieres configurar Safe Exam Browser?
 seb_config_mode_client=Usar la configuraci\u00f3n del cliente
 seb_config_mode_manual=Configurar manualmente
 seb_config_mode_upload=Subir mi propia configuraci\u00f3n
+seb_config_mode_client_help=Using client configuration will allow the use of encrypted or no SEB configuration. In this case no \
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties:423]: @mpellicer Could you add _es translation please?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties [hunk 3]
```diff
@@ -362,6 +362,13 @@ seb_config_mode = Nola konfiguratu nahi duzu Safe Exam Browser?
 seb_config_mode_client = Bezeroaren konfigurazioa erabili
 seb_config_mode_manual = Eskuz konfiguratu
 seb_config_mode_upload = Nire konfigurazioa igo
+seb_config_mode_client_help=Using client configuration will allow the use of encrypted or no SEB configuration. In this case no \
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:365]: Hi @PabloSanRoman, could you add this translations please?

**Review by stetsche — COMMENTED** [review]
For 2. translations of help texts are missing

**Review by mpellicer — APPROVED** [review]

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties (hunk 3 above)
Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:365]: Hi @mpellicer 

+ seb_config_mode_client_help= Bezeroaren konfigurazioa erabiltzeak aukera emango du konfigurazio zifratua erabiltzeko edo SEB gabekoa erabiltzeko. Kasu honetan, Sakaik ez dio konfigurazio fitxategirik emango ikasleari.

Comment (mpellicer) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:365]: Thank you! Included in the S2U-21 work.

**Pull request merged**